packages/web-react/src/types/shared/element.ts

```diff
@@ -19,6 +19,11 @@ export type SpiritPolymorphicElementPropsWithoutRef<E extends ElementType, P> =
   ComponentPropsWithoutRef<E>,
   keyof P
 >;
+/**
+ * For more information about `PolymorphicRef` type:
```

Comment:
```suggestion
 * Type React's forwarded ref with `PolymorphicRef` to allow generic `elementType` to be strongly typed, e.g. component allows switching of elements
```

Reply:
changed, thank you for better description ❤️